server/etcdserver/raft.go

```diff
@@ -157,7 +157,7 @@ func (r *raftNode) tick() {
 // to modify the fields after it has been started.
 func (r *raftNode) start(rh *raftReadyHandler) {
 	internalTimeout := time.Second
-
+	// gofail: var raftBeforeStart struct{}
```

Comment:
This is not needed.

Reply:
I mean that this is not a useful gofail point so let's not add it. If you want to test sleep failpoint you can just use other ones like `backend/defragBeforeCopy`.